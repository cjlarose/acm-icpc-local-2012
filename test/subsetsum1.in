2
5859870 3
3141592
2718281
1000000
335502815 6
735923020
487543581
938712976
239838756
427959134
228185544
