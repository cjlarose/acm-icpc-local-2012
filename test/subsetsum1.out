5859873
427959134
